File: 240520/숫자의 증감/increasing-and-decreasing-numbers.py
r = input()
arr = r.split()
c = str(arr[0])
n = int(arr[1])

if c == "A":
    for i in range(1, n + 1):
        print(i, end=' ')
        i += 1
elif c == "D":
    for i in range(n, 0, -1):
        print(i, end=' ')
        # i -= 1

File: 240825/숫자의 증감/increasing-and-decreasing-numbers.js
// 변수 선언 및 입력
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ");
let c = input[0];
let n = Number(input[1]);

// 출력
let result = "";
if (c === 'A') {
    for (let i = 1; i <= n; i++) {
        result += i + " ";
    }
}
else {
    for (let i = n; i >= 1; i--) {
        result += i + " ";
    }
}
console.log(result);